eg7.py

```diff
@@ -0,0 +1,26 @@
+"""You are given a list of email addresses. Filter out the validate emails."""
+
+import re     #importing the re module to implement regular expression
+
+#defining what should be include in email using re.compile() method referd as email_symbols
+email_symbols = re.compile(r'([A-Za-z0-9_-])+@[A-Za-z0-9]+(\.[A-Z|a-z]{2,3})')
+
+#method for check the comming mail as parameter is valid or not 
+def check_email(email):
+    is_valid=re.fullmatch(email_symbols, email)   #is Valid is filter out the correct email 
+    if is_valid:
+      return True        #if any correct email is found then function will satisfied and return true
+    else:
```

Comment:
You can remove this `else` and directly write return False here. Because if it is True then this line will not even got executed.

Reply:
yes i have removed else.